test/libsolidity/syntaxTests/dataLocations/transient_function_type.sol

```diff
@@ -0,0 +1,4 @@
+contract C {
+	function (uint) external transient y;
```

Comment:
Please include a test case for this too (it should work):
```solidity
	function (uint) external transient;
```

Reply:
Done.